### ISSUE TITLE
AM2R server couldn't recognize when my Mac was in game

### ISSUE THREAD

**Issue by Shock9616** [DodoBirby/AM2R-Multitroid-Unofficial-Patch#6]
I was attempting to set up a co-op session with a friend earlier today, and while I could connect to the server running on my friend's PC (through Hamachi), but we couldn't get the server to recognize that I was in the game. Whenever I would connect it would say `Normal:Not Ready` next to my name on the server no matter what we tried. I asked for help on the AM2R discord server and was told that it should have been working. My friend's game was recognized and he could put himself in SA-X mode and everything, but my game was essentially just single player. This leads me to believe that it is probably an issue with the Mac version of the mod. Issue #3 says that there was no way of testing the Mac build so I think this is plausible, but if there is something extra that needs to be done with the Mac version over the others I would be happy to try it.
![Screen_Shot_2022-07-29_at_5 15 32_PM](https://user-images.githubusercontent.com/28808006/181876054-781e4115-4f7e-4965-af44-e36480f01eef.jpg)

**Comment by Shock9616** [comment]
Quick update, don't know if it's helpful, but I also tried running the server on my Mac (through wine) and the results are still the same. I can connect but the server doesn't recognize when I'm in-game, and my friend's game works fine.

**Comment by DodoBirby** [comment]
Yeah I have no way of testing the Mac builds so it's sort of a blind conversion process. I'll try see if maybe there are certain functions that Mac doesn't support.

**Comment by Shock9616** [comment]
Ok sounds good! Thank you!

**Comment by Shock9616** [comment]
If you want, I would be happy to test the mac builds for you. That way you don't have to cross your fingers and hope they work 🙂
